index.html

```diff
@@ -31,6 +42,7 @@ <h1>RustFest {{site.location.city}} {{site.dates.year}}</h1>
         </p>
       </div>
       <div class="location">
+        <br/>
```

Comment:
why? CSS should handle this

Reply:
right, and that's not even needed so I'll remove it.

However I'll try improving a bit the blocks positioning here, I prefer the one that is showing on the [Paris RustFest](https://paris.rustfest.eu).

(Won't spend too much much time on this, promised)

Reply:
uhm ok probably the paris site has the video container that's pushing upward that `div`. Anyway, this page will be updated at a later stage, so I wont spend too much here and now.